sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -73,6 +78,19 @@ grammar SqlBase;
       return false;
     }
   }
+
+  /**
+   * This method will be called when the character stream ends and try to find out the
+   * unclosed bracketed comment.
+   * If the next character is -1, it means the end of the entire character stream match,
```

Comment:
BTW do we need to check? looking at `('*/' | {end();} EOF)`, seems it's always EOF if we enter this method.

Reply:
Yes